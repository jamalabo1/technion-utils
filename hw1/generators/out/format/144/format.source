C X 273 1
C X 887 2
B Z 969 3
D Y 581 4
C Y 184 5
A Z 129 6
A W 616 7
A X 381 8
C W 461 9
B Y 746 10
C X 368 11
C X 601 12
D Y 978 13
A Z 539 14
C X 453 15
D Y 697 16
A W 452 17
A Z 596 18
C Y 583 19
D X 885 20
B Z 982 21
B W 357 22
C Y 337 23
A W 347 24
B W 48 25
B X 941 26
C X 766 27
A W 436 28
D W 641 29
B W 267 30
D X 911 31
A Z 733 32
B Y 746 33
A W 827 34
A Y 641 35
A X 515 36
A X 973 37
D X 941 38
C X 384 39
B Z 554 40
B W 269 41
C W 739 42
A W 903 43
C W 931 44
A W 593 45
D W 183 46
C W 78 47
A X 943 48
D W 789 49
A W 562 50
A X 612 51
A X 980 52
A Z 857 53
B X 849 54
C W 446 55
B W 15 56
C W 144 57
D W 674 58
B W 960 59
A X 624 60
D W 0 61
B Z 105 62
D X 370 63
A X 402 64
B X 452 65
B X 725 66
B X 390 67
D Y 857 68
A Y 780 69
D W 928 70
D X 763 71
B Z 793 72
A Y 717 73
B Z 671 74
A X 81 75
B Z 416 76
C Z 352 77
D Z 721 78
D Z 468 79
C Z 271 80
B X 723 81
C Y 503 82
B Z 510 83
B W 731 84
D Z 760 85
D Z 476 86
A Z 891 87
B Z 448 88
A Z 630 89
A Y 74 90
A W 225 91
A Y 219 92
C W 707 93
A Y 740 94
C Z 595 95
D X 189 96
A X 612 97
A Y 803 98
C Y 586 99
B Z 141 100
A Z 313 101
D X 459 102
D Z 716 103
D X 992 104
A Z 622 105
A Y 915 106
C Y 887 107
D Z 938 108
A Y 599 109
D W 723 110
C Z 888 111
D W 341 112
B Y 231 113
C W 754 114
C X 3 115
C Z 251 116
D Z 329 117
D Y 288 118
C W 474 119
A Y 858 120
B Y 57 121
B Y 270 122
B Z 938 123
C Z 446 124
D X 570 125
B Y 98 126
D Y 863 127
C X 153 128
C Y 508 129
C Y 593 130
C Y 712 131
D X 578 132
D Y 519 133
A Y 745 134
A X 332 135
B X 615 136
D X 253 137
B X 609 138
C Y 882 139
A Z 123 140
D W 835 141
C W 357 142
D X 259 143
D Y 688 144
A W 993 145
D Y 917 146
A X 795 147
D Z 341 148
B W 339 149
D Z 37 150
A X 981 151
A Z 777 152
D W 542 153
B X 640 154
C Y 441 155
B W 819 156
B W 189 157
B Z 104 158
C Z 894 159
D Z 676 160
B X 732 161