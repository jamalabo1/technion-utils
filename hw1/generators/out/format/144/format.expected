Blockchain Info:
1
Sender Name: B
Receiver Name: X
Transaction Value: 732
Transaction timestamp: 161
2
Sender Name: D
Receiver Name: Z
Transaction Value: 676
Transaction timestamp: 160
3
Sender Name: C
Receiver Name: Z
Transaction Value: 894
Transaction timestamp: 159
4
Sender Name: B
Receiver Name: Z
Transaction Value: 104
Transaction timestamp: 158
5
Sender Name: B
Receiver Name: W
Transaction Value: 189
Transaction timestamp: 157
6
Sender Name: B
Receiver Name: W
Transaction Value: 819
Transaction timestamp: 156
7
Sender Name: C
Receiver Name: Y
Transaction Value: 441
Transaction timestamp: 155
8
Sender Name: B
Receiver Name: X
Transaction Value: 640
Transaction timestamp: 154
9
Sender Name: D
Receiver Name: W
Transaction Value: 542
Transaction timestamp: 153
10
Sender Name: A
Receiver Name: Z
Transaction Value: 777
Transaction timestamp: 152
11
Sender Name: A
Receiver Name: X
Transaction Value: 981
Transaction timestamp: 151
12
Sender Name: D
Receiver Name: Z
Transaction Value: 37
Transaction timestamp: 150
13
Sender Name: B
Receiver Name: W
Transaction Value: 339
Transaction timestamp: 149
14
Sender Name: D
Receiver Name: Z
Transaction Value: 341
Transaction timestamp: 148
15
Sender Name: A
Receiver Name: X
Transaction Value: 795
Transaction timestamp: 147
16
Sender Name: D
Receiver Name: Y
Transaction Value: 917
Transaction timestamp: 146
17
Sender Name: A
Receiver Name: W
Transaction Value: 993
Transaction timestamp: 145
18
Sender Name: D
Receiver Name: Y
Transaction Value: 688
Transaction timestamp: 144
19
Sender Name: D
Receiver Name: X
Transaction Value: 259
Transaction timestamp: 143
20
Sender Name: C
Receiver Name: W
Transaction Value: 357
Transaction timestamp: 142
21
Sender Name: D
Receiver Name: W
Transaction Value: 835
Transaction timestamp: 141
22
Sender Name: A
Receiver Name: Z
Transaction Value: 123
Transaction timestamp: 140
23
Sender Name: C
Receiver Name: Y
Transaction Value: 882
Transaction timestamp: 139
24
Sender Name: B
Receiver Name: X
Transaction Value: 609
Transaction timestamp: 138
25
Sender Name: D
Receiver Name: X
Transaction Value: 253
Transaction timestamp: 137
26
Sender Name: B
Receiver Name: X
Transaction Value: 615
Transaction timestamp: 136
27
Sender Name: A
Receiver Name: X
Transaction Value: 332
Transaction timestamp: 135
28
Sender Name: A
Receiver Name: Y
Transaction Value: 745
Transaction timestamp: 134
29
Sender Name: D
Receiver Name: Y
Transaction Value: 519
Transaction timestamp: 133
30
Sender Name: D
Receiver Name: X
Transaction Value: 578
Transaction timestamp: 132
31
Sender Name: C
Receiver Name: Y
Transaction Value: 712
Transaction timestamp: 131
32
Sender Name: C
Receiver Name: Y
Transaction Value: 593
Transaction timestamp: 130
33
Sender Name: C
Receiver Name: Y
Transaction Value: 508
Transaction timestamp: 129
34
Sender Name: C
Receiver Name: X
Transaction Value: 153
Transaction timestamp: 128
35
Sender Name: D
Receiver Name: Y
Transaction Value: 863
Transaction timestamp: 127
36
Sender Name: B
Receiver Name: Y
Transaction Value: 98
Transaction timestamp: 126
37
Sender Name: D
Receiver Name: X
Transaction Value: 570
Transaction timestamp: 125
38
Sender Name: C
Receiver Name: Z
Transaction Value: 446
Transaction timestamp: 124
39
Sender Name: B
Receiver Name: Z
Transaction Value: 938
Transaction timestamp: 123
40
Sender Name: B
Receiver Name: Y
Transaction Value: 270
Transaction timestamp: 122
41
Sender Name: B
Receiver Name: Y
Transaction Value: 57
Transaction timestamp: 121
42
Sender Name: A
Receiver Name: Y
Transaction Value: 858
Transaction timestamp: 120
43
Sender Name: C
Receiver Name: W
Transaction Value: 474
Transaction timestamp: 119
44
Sender Name: D
Receiver Name: Y
Transaction Value: 288
Transaction timestamp: 118
45
Sender Name: D
Receiver Name: Z
Transaction Value: 329
Transaction timestamp: 117
46
Sender Name: C
Receiver Name: Z
Transaction Value: 251
Transaction timestamp: 116
47
Sender Name: C
Receiver Name: X
Transaction Value: 3
Transaction timestamp: 115
48
Sender Name: C
Receiver Name: W
Transaction Value: 754
Transaction timestamp: 114
49
Sender Name: B
Receiver Name: Y
Transaction Value: 231
Transaction timestamp: 113
50
Sender Name: D
Receiver Name: W
Transaction Value: 341
Transaction timestamp: 112
51
Sender Name: C
Receiver Name: Z
Transaction Value: 888
Transaction timestamp: 111
52
Sender Name: D
Receiver Name: W
Transaction Value: 723
Transaction timestamp: 110
53
Sender Name: A
Receiver Name: Y
Transaction Value: 599
Transaction timestamp: 109
54
Sender Name: D
Receiver Name: Z
Transaction Value: 938
Transaction timestamp: 108
55
Sender Name: C
Receiver Name: Y
Transaction Value: 887
Transaction timestamp: 107
56
Sender Name: A
Receiver Name: Y
Transaction Value: 915
Transaction timestamp: 106
57
Sender Name: A
Receiver Name: Z
Transaction Value: 622
Transaction timestamp: 105
58
Sender Name: D
Receiver Name: X
Transaction Value: 992
Transaction timestamp: 104
59
Sender Name: D
Receiver Name: Z
Transaction Value: 716
Transaction timestamp: 103
60
Sender Name: D
Receiver Name: X
Transaction Value: 459
Transaction timestamp: 102
61
Sender Name: A
Receiver Name: Z
Transaction Value: 313
Transaction timestamp: 101
62
Sender Name: B
Receiver Name: Z
Transaction Value: 141
Transaction timestamp: 100
63
Sender Name: C
Receiver Name: Y
Transaction Value: 586
Transaction timestamp: 99
64
Sender Name: A
Receiver Name: Y
Transaction Value: 803
Transaction timestamp: 98
65
Sender Name: A
Receiver Name: X
Transaction Value: 612
Transaction timestamp: 97
66
Sender Name: D
Receiver Name: X
Transaction Value: 189
Transaction timestamp: 96
67
Sender Name: C
Receiver Name: Z
Transaction Value: 595
Transaction timestamp: 95
68
Sender Name: A
Receiver Name: Y
Transaction Value: 740
Transaction timestamp: 94
69
Sender Name: C
Receiver Name: W
Transaction Value: 707
Transaction timestamp: 93
70
Sender Name: A
Receiver Name: Y
Transaction Value: 219
Transaction timestamp: 92
71
Sender Name: A
Receiver Name: W
Transaction Value: 225
Transaction timestamp: 91
72
Sender Name: A
Receiver Name: Y
Transaction Value: 74
Transaction timestamp: 90
73
Sender Name: A
Receiver Name: Z
Transaction Value: 630
Transaction timestamp: 89
74
Sender Name: B
Receiver Name: Z
Transaction Value: 448
Transaction timestamp: 88
75
Sender Name: A
Receiver Name: Z
Transaction Value: 891
Transaction timestamp: 87
76
Sender Name: D
Receiver Name: Z
Transaction Value: 476
Transaction timestamp: 86
77
Sender Name: D
Receiver Name: Z
Transaction Value: 760
Transaction timestamp: 85
78
Sender Name: B
Receiver Name: W
Transaction Value: 731
Transaction timestamp: 84
79
Sender Name: B
Receiver Name: Z
Transaction Value: 510
Transaction timestamp: 83
80
Sender Name: C
Receiver Name: Y
Transaction Value: 503
Transaction timestamp: 82
81
Sender Name: B
Receiver Name: X
Transaction Value: 723
Transaction timestamp: 81
82
Sender Name: C
Receiver Name: Z
Transaction Value: 271
Transaction timestamp: 80
83
Sender Name: D
Receiver Name: Z
Transaction Value: 468
Transaction timestamp: 79
84
Sender Name: D
Receiver Name: Z
Transaction Value: 721
Transaction timestamp: 78
85
Sender Name: C
Receiver Name: Z
Transaction Value: 352
Transaction timestamp: 77
86
Sender Name: B
Receiver Name: Z
Transaction Value: 416
Transaction timestamp: 76
87
Sender Name: A
Receiver Name: X
Transaction Value: 81
Transaction timestamp: 75
88
Sender Name: B
Receiver Name: Z
Transaction Value: 671
Transaction timestamp: 74
89
Sender Name: A
Receiver Name: Y
Transaction Value: 717
Transaction timestamp: 73
90
Sender Name: B
Receiver Name: Z
Transaction Value: 793
Transaction timestamp: 72
91
Sender Name: D
Receiver Name: X
Transaction Value: 763
Transaction timestamp: 71
92
Sender Name: D
Receiver Name: W
Transaction Value: 928
Transaction timestamp: 70
93
Sender Name: A
Receiver Name: Y
Transaction Value: 780
Transaction timestamp: 69
94
Sender Name: D
Receiver Name: Y
Transaction Value: 857
Transaction timestamp: 68
95
Sender Name: B
Receiver Name: X
Transaction Value: 390
Transaction timestamp: 67
96
Sender Name: B
Receiver Name: X
Transaction Value: 725
Transaction timestamp: 66
97
Sender Name: B
Receiver Name: X
Transaction Value: 452
Transaction timestamp: 65
98
Sender Name: A
Receiver Name: X
Transaction Value: 402
Transaction timestamp: 64
99
Sender Name: D
Receiver Name: X
Transaction Value: 370
Transaction timestamp: 63
100
Sender Name: B
Receiver Name: Z
Transaction Value: 105
Transaction timestamp: 62
101
Sender Name: D
Receiver Name: W
Transaction Value: 0
Transaction timestamp: 61
102
Sender Name: A
Receiver Name: X
Transaction Value: 624
Transaction timestamp: 60
103
Sender Name: B
Receiver Name: W
Transaction Value: 960
Transaction timestamp: 59
104
Sender Name: D
Receiver Name: W
Transaction Value: 674
Transaction timestamp: 58
105
Sender Name: C
Receiver Name: W
Transaction Value: 144
Transaction timestamp: 57
106
Sender Name: B
Receiver Name: W
Transaction Value: 15
Transaction timestamp: 56
107
Sender Name: C
Receiver Name: W
Transaction Value: 446
Transaction timestamp: 55
108
Sender Name: B
Receiver Name: X
Transaction Value: 849
Transaction timestamp: 54
109
Sender Name: A
Receiver Name: Z
Transaction Value: 857
Transaction timestamp: 53
110
Sender Name: A
Receiver Name: X
Transaction Value: 980
Transaction timestamp: 52
111
Sender Name: A
Receiver Name: X
Transaction Value: 612
Transaction timestamp: 51
112
Sender Name: A
Receiver Name: W
Transaction Value: 562
Transaction timestamp: 50
113
Sender Name: D
Receiver Name: W
Transaction Value: 789
Transaction timestamp: 49
114
Sender Name: A
Receiver Name: X
Transaction Value: 943
Transaction timestamp: 48
115
Sender Name: C
Receiver Name: W
Transaction Value: 78
Transaction timestamp: 47
116
Sender Name: D
Receiver Name: W
Transaction Value: 183
Transaction timestamp: 46
117
Sender Name: A
Receiver Name: W
Transaction Value: 593
Transaction timestamp: 45
118
Sender Name: C
Receiver Name: W
Transaction Value: 931
Transaction timestamp: 44
119
Sender Name: A
Receiver Name: W
Transaction Value: 903
Transaction timestamp: 43
120
Sender Name: C
Receiver Name: W
Transaction Value: 739
Transaction timestamp: 42
121
Sender Name: B
Receiver Name: W
Transaction Value: 269
Transaction timestamp: 41
122
Sender Name: B
Receiver Name: Z
Transaction Value: 554
Transaction timestamp: 40
123
Sender Name: C
Receiver Name: X
Transaction Value: 384
Transaction timestamp: 39
124
Sender Name: D
Receiver Name: X
Transaction Value: 941
Transaction timestamp: 38
125
Sender Name: A
Receiver Name: X
Transaction Value: 973
Transaction timestamp: 37
126
Sender Name: A
Receiver Name: X
Transaction Value: 515
Transaction timestamp: 36
127
Sender Name: A
Receiver Name: Y
Transaction Value: 641
Transaction timestamp: 35
128
Sender Name: A
Receiver Name: W
Transaction Value: 827
Transaction timestamp: 34
129
Sender Name: B
Receiver Name: Y
Transaction Value: 746
Transaction timestamp: 33
130
Sender Name: A
Receiver Name: Z
Transaction Value: 733
Transaction timestamp: 32
131
Sender Name: D
Receiver Name: X
Transaction Value: 911
Transaction timestamp: 31
132
Sender Name: B
Receiver Name: W
Transaction Value: 267
Transaction timestamp: 30
133
Sender Name: D
Receiver Name: W
Transaction Value: 641
Transaction timestamp: 29
134
Sender Name: A
Receiver Name: W
Transaction Value: 436
Transaction timestamp: 28
135
Sender Name: C
Receiver Name: X
Transaction Value: 766
Transaction timestamp: 27
136
Sender Name: B
Receiver Name: X
Transaction Value: 941
Transaction timestamp: 26
137
Sender Name: B
Receiver Name: W
Transaction Value: 48
Transaction timestamp: 25
138
Sender Name: A
Receiver Name: W
Transaction Value: 347
Transaction timestamp: 24
139
Sender Name: C
Receiver Name: Y
Transaction Value: 337
Transaction timestamp: 23
140
Sender Name: B
Receiver Name: W
Transaction Value: 357
Transaction timestamp: 22
141
Sender Name: B
Receiver Name: Z
Transaction Value: 982
Transaction timestamp: 21
142
Sender Name: D
Receiver Name: X
Transaction Value: 885
Transaction timestamp: 20
143
Sender Name: C
Receiver Name: Y
Transaction Value: 583
Transaction timestamp: 19
144
Sender Name: A
Receiver Name: Z
Transaction Value: 596
Transaction timestamp: 18
145
Sender Name: A
Receiver Name: W
Transaction Value: 452
Transaction timestamp: 17
146
Sender Name: D
Receiver Name: Y
Transaction Value: 697
Transaction timestamp: 16
147
Sender Name: C
Receiver Name: X
Transaction Value: 453
Transaction timestamp: 15
148
Sender Name: A
Receiver Name: Z
Transaction Value: 539
Transaction timestamp: 14
149
Sender Name: D
Receiver Name: Y
Transaction Value: 978
Transaction timestamp: 13
150
Sender Name: C
Receiver Name: X
Transaction Value: 601
Transaction timestamp: 12
151
Sender Name: C
Receiver Name: X
Transaction Value: 368
Transaction timestamp: 11
152
Sender Name: B
Receiver Name: Y
Transaction Value: 746
Transaction timestamp: 10
153
Sender Name: C
Receiver Name: W
Transaction Value: 461
Transaction timestamp: 9
154
Sender Name: A
Receiver Name: X
Transaction Value: 381
Transaction timestamp: 8
155
Sender Name: A
Receiver Name: W
Transaction Value: 616
Transaction timestamp: 7
156
Sender Name: A
Receiver Name: Z
Transaction Value: 129
Transaction timestamp: 6
157
Sender Name: C
Receiver Name: Y
Transaction Value: 184
Transaction timestamp: 5
158
Sender Name: D
Receiver Name: Y
Transaction Value: 581
Transaction timestamp: 4
159
Sender Name: B
Receiver Name: Z
Transaction Value: 969
Transaction timestamp: 3
160
Sender Name: C
Receiver Name: X
Transaction Value: 887
Transaction timestamp: 2
161
Sender Name: C
Receiver Name: X
Transaction Value: 273
Transaction timestamp: 1